model/Operations/Operations.md

```diff
@@ -0,0 +1,22 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Operations
+
+## Summary
+
+The Operations Profile defines fields for describing the business context of the software that cannot (or not yet) be directly extracted from the source repository. Operations include technical operations and business operations.
```

Comment:
```suggestion
The Operations profile defines fields for describing the business context of the software that cannot (or not yet) be directly extracted from the source repository. Operations include technical operations and business operations.
```

Use lowercase "profile", according to https://github.com/spdx/spdx-spec/issues/1250